Sources/MapboxCoreNavigation/EventDetails.swift

```diff
@@ -18,13 +18,15 @@ struct PerformanceEventDetails: EventDetails {
     var event: String?
     var counters: [Counter] = []
     var attributes: [Attribute] = []
+    var appMetadata: [String: String?]?
```

Comment:
Maybe it worth placing this property to the protocol as it's common for every navigation event

Reply:
Ah it is in the NavigationEventDetails protocol

Reply:
Is there any implementation of the `EventDetails` protocol without this field?